hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/MaintainersEndpoint.java

```diff
@@ -0,0 +1,101 @@
+package pl.allegro.tech.hermes.management.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.wordnik.swagger.annotations.Api;
+import com.wordnik.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import pl.allegro.tech.hermes.api.ErrorCode;
+import pl.allegro.tech.hermes.api.Maintainer;
+import pl.allegro.tech.hermes.common.exception.HermesException;
+import pl.allegro.tech.hermes.management.domain.maintainer.HintingMaintainerSource;
+import pl.allegro.tech.hermes.management.domain.maintainer.MaintainerSource;
+import pl.allegro.tech.hermes.management.domain.maintainer.MaintainerSourceNotFound;
+import pl.allegro.tech.hermes.management.domain.maintainer.MaintainerSources;
+
+import javax.ws.rs.*;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- javax\.ws\.rs\.\*\.